docs/bplan_api.md

```diff
@@ -4,167 +4,267 @@ mein.berlin provides an external REST API to create and manage BPLAN projects.
+}
+```
+
+## Testing
+
+### Example for `curl`
 
 ```bash
 curl \
  -X POST https://mein.berlin.de/api/organisations/5/bplan/ \
  --user 'test@example.com':'password' \
  -H "Content-Type: application/json" \
  -d \
- "
+ '
  {
-   \"name\":\"Luisenblock Ost - Bebauungsplan 1-70\",
-   \"identifier\": \"VI - 96a\", \"description\": \"Test\",
-   \"url\": \"https://mein.berlin.de\",
-   \"office_worker_email\": \"test@example.com\",
-   \"start_date\": \"2019-01-01T00:00\",
-   \"end_date\": \"2022-01-01T00:00\"
+   "name":"Luisenblock Ost - Bebauungsplan 1-70",
+   "identifier": "VI - 96a",
+   "description": "Test",
+   "url": "https://mein.berlin.de",
+   "office_worker_email": "test@example.com",
+   "start_date": "2019-01-01T00:00",
+   "end_date": "2022-01-01T00:00",
+   "point": {"type": "Feature","geometry": {"type": "Point", "coordinates":[13.411924777644563,52.499598134440944]}}
  }
-"
+'
 ```
 
-#### Example for `httpie`
-```bash
-http \
-   -a test@example:password \
-   -f POST https://mein.berlin.de/api/organisations/5/bplan/ \
-   name="Luisenblock Ost - Bebauungsplan 1-70" \
-   identifier="VI - 96a" \
-   description="Test" \
-   url="https://mein.berlin.de" \
-   office_worker_email="test@example.com" \
-   start_date="2019-01-01T00:00" \
-   end_date="2022-01-01T00:00"
-```
+### Example for `curl` and local testing
 
-#### Example for `httpie` and local testing
 ```bash
-http \
-   -a admin@liqd.net:password \
-   -f POST http://127.0.0.1:8003/api/organisations/1/bplan/ \
-   name="Luisenblock Ost - Bebauungsplan 1-70" \
-   identifier="VI - 96a" \
-   description="Test" \
-   url="https://mein.berlin.de" \
-   office_worker_email="test@example.com" \
-   start_date="2019-01-01T00:00" \
-   end_date="2022-01-01T00:00"
+curl  -X POST http://127.0.0.1:8003/api/organisations/1/bplan/ \
+ --user 'admin@liqd.net':'password' \
+ -H "Content-Type: application/json" \
+ -d \
+ '
+ {
+   "name":"Luisenblock Ost - Bebauungsplan 1-70",
+   "identifier": "VI - 96a",
+   "description": "Test",
+   "url": "https://mein.berlin.de",
+   "office_worker_email": "test@example.com",
+   "start_date": "2019-01-01T00:00",
+   "end_date": "2022-01-01T00:00",
+   "point": {"type": "Feature","geometry": {"type": "Point", "coordinates":[13.411924777644563,52.499598134440944]}}
+ }
+'
 ```
 
-The participation will start end end automatically at the scheduled time.
+## Updating a Bplan
 
-## Updating a BPLAN Project
+Update an existing Bplan with the id `bplan-id`  within the organisation designated by`organisation-id`.
 
-To update a BPLAN setting send a PATCH request with the field to update using
-the *id* of the BPLAN from the create response. If you update an archived BPLAN it will only be unarchived if you set the end_date to a date in the future. Archived BPLANS are not shown in any lists on the platform.
+**URL** : `https://mein.berlin.de/api/organisations/<organisation-id>/bplan/<bplan-id>/1
 
-API endpoint for PATCH:
+**Method** : `PATCH`
```

Comment:
and PUT?

Reply:
I changed the endpoint above, we only support `PATCH`